src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -239,11 +239,11 @@ class SingleActiveFIMap extends React.Component<
     };
     const namePaths =
       plan.jurisdiction_name_path instanceof Array ? plan.jurisdiction_name_path : [];
-    const pages = namePaths.map(namePath => {
+    const pages = namePaths.map((namePath, i) => {
       // return a page object for each name path
       return {
         label: namePath,
-        url: '',
+        url: FI_URL + '/filter/' + plan.jurisdiction_path[i] + '/' + plan.id,
```

Comment:
Please don't use string concatenation here.  Check elsewhere in the code for what we have been consistently using.

Reply:
cool, I see [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are used.

Reply:
Should we put this in a getter function? (which uses a Template Literal to build the string)